test/src/components/serviceInsights/serviceInsights.spec.jsx

```diff
@@ -116,5 +117,28 @@ describe('<ServiceInsights/>', () => {
 
             sinon.assert.calledWith(timeWindow.toTimeRange, '888');
         });
+
+        it('shoudl render an error message when no traces found', () => {
+            const search = {
+                serviceName: 'web-ui',
+                time: {
+                    preset: 'foo'
+                }
+            };
+
+            window.haystackUiConfig.tracesTimePresetOptions = [
+                {
+                    shortName: 'foo',
+                    value: '888'
+                }
+            ];
+
+            const store = createStoreStub('fulfilled', {
+                nodes: []
+            });
+            let wrapper = shallow(<ServiceInsights search={search} store={store} />);
+
+            expect(wrapper.find(Error)).to.have.length(1);
```

Comment:
You should be able to do a find on the string `'Error'` instead of needing to import the component here. I've had luck with that.

Reply:
Works!  Pending here: https://github.com/nathanwalther/haystack-ui/pull/3